homeassistant/components/vesync/const.py

```diff
@@ -4,10 +4,25 @@
 VS_DISCOVERY = "vesync_discovery_{}"
 SERVICE_UPDATE_DEVS = "update_devices"
 
+UPDATE_INTERVAL = 60
+"""
+Update interval for DataCoordinator.
+
+The vesync daily quota formula is 3200 + 1500 * device_count.
+
+An interval of 60 seconds amounts 1440 calls/day which
+would be below the 4700 daily quota. For 2 devices, the
+total would be 2880.
+
+Using 30 seconds interval gives 8640 for 3 devices which
+exceeds the quota of 7700.
+"""
```

Comment:
Why don't we just make this dynamic? We know how many devices someone has so we can calculate how many API calls we can make.

But this would be for a followup

Reply:
Sounds good